src/pages/layout/index.tsx

```diff
@@ -64,7 +67,7 @@ function walletsForNetwork(network: string) {
     openBlockWallet,
     tokenPocketWallet,
     trustWallet,
-    welldoneWallet,
+    welldoneWallet
```

Comment:
I thought it was json style to not have trailing commas?

Reply:
Except that it's Javascript and not JSON.  So you can.